app/src/main/java/org/simple/clinic/crash/SentryCrashReporter.kt

```diff
@@ -64,19 +69,19 @@ class SentryCrashReporter @Inject constructor(
         .map { it.get().isoCountryCode }
         .subscribe(
             {
-              Sentry.getContext().addTag("countryCode", it.toString())
+              Sentry.setTag("countryCode", it.toString())
             },
             { report(it) })
   }
 
   override fun dropBreadcrumb(breadcrumb: Breadcrumb) {
-    val sentryBreadcrumb = BreadcrumbBuilder()
-        .setLevel(priorityToLevel(breadcrumb.priority))
-        .setCategory(breadcrumb.tag)
-        .setMessage(breadcrumb.message)
-        .setTimestamp(Date(System.currentTimeMillis()))
-        .build()
-    Sentry.getContext().recordBreadcrumb(sentryBreadcrumb)
+    val sentryBreadCrumb = SentryBreadcrumb().apply {
+      level = priorityToLevel(breadcrumb.priority)
+      category = breadcrumb.tag
+      message = breadcrumb.message
```

Comment:
Is the timestamp set automatically by Sentry now?

Reply:
Yep